### PR TITLE
#160609522 Like and dislike articles

### PR DESCRIPTION
**Pull request by RonKbS**
#### What does this PR do?
Allow users to like or dislike an article.

#### Description of Task to be completed?
Currently, a user is unable to add a reaction to an article.

This PR allows a user to like or dislike an article. They can add their reactions by using the buttons displayed on the articles.

#### How should this be manually tested?

-Run the application using `npm start`
-Using a logged in user, click on the thumbs up or down button to like and dislike an
article respectively.
-To remove the reactions, use the same buttons.

#### What are the relevant pivotal tracker stories?

[#160609522](https://www.pivotaltracker.com/story/show/160609522)

#### Any background context you want to add?
N/A

## Checklist:

- [x] My IDE is configured to follow the [**ESlint Conventions**](https://github.com/eslint/eslint)

- [x] I have **added tests** to cover my changes.

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the develop branch.

### CODE REVIEW

File: src/config/axiosInstance.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import axios from 'axios';
 
-const baseURL = 'https://ah-backend-targaryen-staging.herokuapp.com';
+const baseURL = 'http://127.0.0.1:8000/';
```
Comment (MuhanguziDavid) [inline src/config/axiosInstance.js]: @RonKbS , please use the heroku link as your `baseURL`, not localhost

File: src/actions/articleActions.js [hunk 2]
```diff
@@ -34,3 +36,37 @@ export const postArticle = postData => dispatch => {
...
+  toast.dismiss();
+  return axiosInstance
+    .post(`/api/articles/${slug}/reaction/`, payload)
+    .then(response => {
+      dispatch(likeDislikeSuccess(true));
+      toast.success(
+        response.data.Message,
+        { autoClose: 3500, hideProgressBar: true },
+        {
+          position: toast.POSITION.TOP_CENTER,
```
Comment (MuhanguziDavid) [inline src/actions/articleActions.js]: @RonKbS , This line is not required, the position for all toasts was already specified.

File: src/components/Articles/LikeDislike.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+// import { Redirect } from 'react-router-dom';
```
Comment (MuhanguziDavid) [inline src/components/Articles/LikeDislike.js]: Please, remove commented out code

**Review by MuhanguziDavid — APPROVED** [review]
LGTM

**Review by samuelbwambale — APPROVED** [review]
@RonKbS Good Job. LGTM

**Pull request merged**